src/overlays/actors/ovl_Boss_03/z_boss_03.c

```diff
@@ -2196,24 +2196,42 @@ s32 Boss03_OverrideLimbDraw(PlayState* play, s32 limbIndex, Gfx** dList, Vec3f*
  * Since there are two sets of ColliderJntSph, indices < 2 (ARRAY_COUNT(sHeadJntSphElementsInit)) refers to the first
  * collider and indices >= 2 refers to the second one
  */
-s8 sGyorgSphElementIndices[] = {
-    -1, // GYORG_LIMB_NONE,
-    -1, // GYORG_LIMB_ROOT,
-    0,  // GYORG_LIMB_HEAD,
-    -1, // GYORG_LIMB_BODY_ROOT,
-    4,  // GYORG_LIMB_UPPER_TRUNK,
-    5,  // GYORG_LIMB_LOWER_TRUNK,
-    6,  // GYORG_LIMB_TAIL,
-    -1, // GYORG_LIMB_RIGHT_FIN_ROOT,
-    2,  // GYORG_LIMB_UPPER_RIGHT_FIN,
-    -1, // GYORG_LIMB_LOWER_RIGHT_FIN,
-    -1, // GYORG_LIMB_LEFT_FIN_ROOT,
-    3,  // GYORG_LIMB_UPPER_LEFT_FIN,
-    -1, // GYORG_LIMB_LOWER_LEFT_FIN,
-    -1, // GYORG_LIMB_JAW_ROOT,
-    1,  // GYORG_LIMB_JAW,
-    -1, // GYORG_LIMB_MAX
-    -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1,
+s8 sGyorgSphElementIndices[2][GYORG_LIMB_MAX] = {
+    {
+        -1, // GYORG_LIMB_NONE
+        -1, // GYORG_LIMB_ROOT
+        0,  // GYORG_LIMB_HEAD
+        -1, // GYORG_LIMB_BODY_ROOT
+        4,  // GYORG_LIMB_UPPER_TRUNK
+        5,  // GYORG_LIMB_LOWER_TRUNK
+        6,  // GYORG_LIMB_TAIL
+        -1, // GYORG_LIMB_RIGHT_FIN_ROOT
+        2,  // GYORG_LIMB_UPPER_RIGHT_FIN
+        -1, // GYORG_LIMB_LOWER_RIGHT_FIN
+        -1, // GYORG_LIMB_LEFT_FIN_ROOT
+        3,  // GYORG_LIMB_UPPER_LEFT_FIN
+        -1, // GYORG_LIMB_LOWER_LEFT_FIN
+        -1, // GYORG_LIMB_JAW_ROOT
+        1,  // GYORG_LIMB_JAW
+    },
+    // unused
+    {
+        -1, // GYORG_LIMB_NONE
+        -1, // GYORG_LIMB_ROOT
+        -1, // GYORG_LIMB_HEAD
+        -1, // GYORG_LIMB_BODY_ROOT
+        -1, // GYORG_LIMB_UPPER_TRUNK
+        -1, // GYORG_LIMB_LOWER_TRUNK
+        -1, // GYORG_LIMB_TAIL
+        -1, // GYORG_LIMB_RIGHT_FIN_ROOT
+        -1, // GYORG_LIMB_UPPER_RIGHT_FIN
+        -1, // GYORG_LIMB_LOWER_RIGHT_FIN
+        -1, // GYORG_LIMB_LEFT_FIN_ROOT
+        -1, // GYORG_LIMB_UPPER_LEFT_FIN
+        -1, // GYORG_LIMB_LOWER_LEFT_FIN
+        -1, // GYORG_LIMB_JAW_ROOT
+        -1, // GYORG_LIMB_JAW
+    },
 };
```

Comment:
I think this seems more reasonable but could use a second thought

Reply:
Seems fine to me